test/system/271-tcp-cors-server.bats

```diff
@@ -8,21 +8,28 @@ load helpers.network
 
 SERVICE_NAME="podman_test_$(random_string)"
 
-SERVICE_TCP_HOST="localhost"
+SERVICE_TCP_HOST="127.0.0.1"
 
 SERVICE_FILE="$UNIT_DIR/$SERVICE_NAME.service"
 SOCKET_FILE="$UNIT_DIR/$SERVICE_NAME.socket"
 
 # bats test_tags=ci:parallel
 @test "podman system service - tcp CORS" {
     skip_if_remote "system service tests are meaningless over remote"
-    PORT=$(random_free_port 63000-64999)
+    PORT=$(random_free_port)
+
     log=${PODMAN_TMPDIR}/system-service.log
     $PODMAN system service --cors="*" tcp:$SERVICE_TCP_HOST:$PORT -t 20 2> $log &
     podman_pid="$!"
-    sleep 5s
-    run curl -s --max-time 10 -vvv $SERVICE_TCP_HOST:$PORT/_ping 2>&1
-    is "$output" ".*< Access-Control-Allow-Origin: \*.*" "access-control-allow-origin verifies CORS is set"
+
+    wait_for_port $SERVICE_TCP_HOST $PORT
+    cmd="curl -s --max-time 10 -vvv $SERVICE_TCP_HOST:$PORT/_ping"
+    echo "$_LOG_PROMPT $cmd"
+    run $cmd
```

Comment:
Should this verify the curl exit code is 0?

Reply:
OOPS! Yes, of course. Sometimes I hate bats.